src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -337,3 +337,5 @@ export const CHAR_LIMITS = [
 
 // migration max string length
 export const MAX_HOUSING_STRING_LENGTH = 500;
+
+export const OMB_CONTROL = '2900-0747';
```

Comment:
Just curious, what is the OMB number?

Reply:
It's a unique number assigned to forms. You can see it on the top right corner of 1st page of the paper/pdf version of 526. Also definition...

> OMB Numbers are form and information collection numbers assigned by The Office of Management and Budget. The Office of Management and Budget is the largest office within the Executive Office of the President of the United States. The OMB generates these Control Numbers under the Paperwork Reduction Act of 1980.

Reply:
Ahh gotcha! Thanks!